packages/cerebral/src/Controller.js

```diff
@@ -67,7 +68,7 @@ class Controller extends EventEmitter {
   */
   flush () {
     const changes = this.model.flush()
-    const computedsAboutToBecomeDirty = computedDependencyStore.getUniqueEntities(changes)
+    const computedsAboutToBecomeDirty = this.strictRender ? computedDependencyStore.getStrictUniqueEntities(changes) : computedDependencyStore.getUniqueEntities(changes)
```

Comment:
Yeah, though do not like that format... hehe... just keep as it is now? :-) It just reads weird to me that:

```js
this.strictRender
? ...
``` 

Cause the first line does not indicate anything, it looks like a syntax error, compared to:

```js
this.strictRender ?
  ...
```

It just indicates what follows :) Anyways, opinions... just keep as is now?